app/models/katello/content.rb

```diff
@@ -54,5 +54,13 @@ def self.import_all
         prod_content_importer.import
       end
     end
+
+    def can_update_to_url?(new_url)
```

Comment:
Are we getting a guarantee that any time a repo url changes the same substitutions will be there? Otherwise we're going to run into problems with this restriction, right?

Reply:
Thats the arrangement. Same substitutions.